apps/web/components/shared/kyc-4-upload.tsx

```
@@ -0,0 +1,465 @@
+      </CardHeader>
+      
+      <CardContent className="space-y-6">
+        <div>
+          <h3 className="text-lg font-medium mb-2">Step 4 of 4</h3>
+          <div className="w-full h-2 bg-gray-200 rounded-full">
+            <div className="w-full h-2 bg-black rounded-full"></div>
+          </div>
+        </div>
+
+        <div className="space-y-2">
+          <label className="text-lg font-medium">Document Type</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)